### PR TITLE
auto for case OCP-10643

### PR DESCRIPTION
**Pull request by zhouying7780**
auto for case OCP-10643

@xingxingxia  Please help review , thanks.

### CODE REVIEW

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
...
 		})
 	})
+
+	g.Describe("when no triggers are defined [Conformance]", func() {
+		dcName := "paused"
+		g.AfterEach(func() {
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should manually make deploy", func() {
```
Comment (xingxingxia) [inline test/extended/deployments/deployments.go]: In upstream contribution it is better to use better syntax. I suggest the verb "deploy" be corrected to a noun "deployment"

File: test/extended/deployments/deployments.go [hunk 2]
```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
...
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(dc.Name).To(o.Equal(dcName))
+
+			g.By("remove all the triggers")
+			_, err = oc.Run("set").Args("triggers", "dc/"+dcName, "--remove-all").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("resume the DC")
+			_, err = oc.Run("rollout").Args("resume", "dc/"+dcName).Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
```
Comment (xingxingxia) [inline test/extended/deployments/deployments.go]: Can we choose test data other than pausedDeploymentFixture so we can reduce the "resume" operation in this test?

**Comment by xingxingxia** [comment]
The PR generally LGTM from peer review. Did you check no existing code already covers your PR? If yes, then could @ Workloads Dev to review/merge. Thanks

File: test/extended/deployments/deployments.go (hunk 1 above)
Comment (zhouying7780) [reply test/extended/deployments/deployments.go]: Updated.

File: test/extended/deployments/deployments.go (hunk 2 above)
Comment (zhouying7780) [reply test/extended/deployments/deployments.go]: Sorry, If we updated with other test data , we need to add other steps to check the non-auto deployment. So this one is appropriate。

Comment (xingxingxia) [reply test/extended/deployments/deployments.go]: OK

File: test/extended/deployments/deployments.go [hunk 3]
```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
...
+			failureTrap(oc, dcName, g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should manually make deployment", func() {
+			dc, err := createDeploymentConfig(oc, pausedDeploymentFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(dc.Name).To(o.Equal(dcName))
+
+			g.By("remove all the triggers")
+			_, err = oc.Run("set").Args("triggers", "dc/"+dcName, "--remove-all").Output()
```
Comment (tnozicka) [inline test/extended/deployments/deployments.go]: you can just create a fixture with `triggers: []`

Comment (tnozicka) [reply test/extended/deployments/deployments.go]: prefer using api call unless you'd be explicitly testing cli

File: test/extended/deployments/deployments.go [hunk 4]
```diff
@@ -1554,4 +1554,42 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
...
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("resume the DC")
+			_, err = oc.Run("rollout").Args("resume", "dc/"+dcName).Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			g.By("check the DC")
```
Comment (tnozicka) [inline test/extended/deployments/deployments.go]: try being more descriptive to what property you are checking

Comment (xingxingxia) [reply test/extended/deployments/deployments.go]: Good point :+1: , in **upstream** code contribution, we should take stricter care of sentences, syntax etc.

File: test/extended/deployments/deployments.go (hunk 3 above)
Comment (zhouying7780) [reply test/extended/deployments/deployments.go]: Thanks , updated.

**Comment by zhouying7780** [comment]
For the confilicts will close this pr and open a new one.

**Pull request closed without merging**